js_modules/dagit/packages/core/src/pipelines/GraphExplorer.tsx

```diff
@@ -138,8 +138,8 @@ export const GraphExplorer: React.FC<GraphExplorerProps> = (props) => {
   );
 
   const {all} = queryResultOps;
-  const highlightedOps = React.useMemo(() => all.filter((s) => s.name.includes(highlighted)), [
-    highlighted,
+  const highlightedOps = React.useMemo(() => all.filter((s) => s.name.includes(nameMatch)), [
```

Comment:
Probably doesn't matter too much, but I wonder if this should be done on `toLowerCase`'d strings.

Reply:
seems like it can't hurt, I'll change this real quick. I'm actually not sure we have ever tested with camel cased or capitalized op / asset names, I bet some of our URLs are pretty case sensitive!